build.gradle

```diff
@@ -11,8 +11,8 @@ plugins {
 	id 'eclipse'
 	id 'maven-publish'
 	id 'org.quiltmc.gradle.licenser' version '1.1.+'
-	id 'org.quiltmc.loom' version '0.12.+' apply false
-	id 'com.github.johnrengelman.shadow' version '7.0.0'
+	id 'org.quiltmc.loom' version '0.13.+' apply false
```

Comment:
0.13 isn't really supported due to it being essentially a beta to 1.0; Could you try that one?

Reply:
Loom 0.13 seems to be working. (so far)

Reply:
What she's saying is to use 1.0 or 0.12, but not 0.13.

Reply:
Undord

Reply:
I'm still curious about the avoidance of Loom - [ ] 1.0; Are there are major errors or?